presto-hive/src/main/java/io/prestosql/plugin/hive/HiveWriterFactory.java

```diff
@@ -522,7 +523,7 @@ else if (insertExistingPartitionsBehavior == InsertExistingPartitionsBehavior.ER
             }
 
             List<Type> types = dataColumns.stream()
-                    .map(column -> column.getHiveType().getType(typeManager))
+                    .map(column -> column.getHiveType().getType(typeManager, getTimestampPrecision(session).getPrecision()))
```

Comment:
`getType(TypeManager)` overload is still used; for example `io.prestosql.plugin.hive.HiveMetadata#finishCreateTable`, `io.prestosql.plugin.hive.HiveMetadata#finishInsert` call sites look relevant 

please make sure there is an issue covering cleanup of the other ones